Source/ExecutionEngine/CommandLineOptions.cs

```diff
@@ -1978,7 +1982,10 @@ similar effects.
                 Limit the number of seconds spent trying to verify
                 each procedure
   /rlimit:<num>
```

Comment:
Please update the documentation to say that this option is deprecated and will be removed.